Monal/Classes/ChatView.swift

```diff
@@ -378,6 +407,19 @@ struct ChatView: View {
                 }
             }
         }
+        .onReceive(NotificationCenter.default.publisher(for: NSNotification.Name("kMonalNewMessageNotice")).receive(on: RunLoop.main)) { notification in
+            DDLogVerbose("chat view got new message notice \(notification.userInfo)")
```

Comment:
This produces the warning `expression implicitly coerced from '[AnyHashable : Any]?' to 'Any'
`. Use this instead:

```suggestion
            DDLogVerbose("chat view got new message notice \(String(describing:notification.userInfo))")
```